Mathlib/Data/Nat/Nth.lean

```diff
@@ -237,10 +237,21 @@ theorem nth_eq_zero {n} :
   · rintro (⟨h₀, rfl⟩ | ⟨hf, hle⟩)
     exacts [nth_zero_of_zero h₀, nth_of_card_le hf hle]
 
+lemma lt_toFinset_card_of_nth_ne_zero {n : ℕ} (h : nth p n ≠ 0) (hf : (setOf p).Finite) :
```

Comment:
Hmm, I think we might want `_card_toFinset`, since `card` is the outer part. Unless we make an exception for `card`?

Reply:
I've reversed the order and also changed it to use the `#` notation for `card`.